AutomatedTesting/Gem/PythonTests/Terrain/TestSuite_Main.py

```diff
@@ -19,7 +19,9 @@
 @pytest.mark.parametrize("launcher_platform", ['windows_editor'])
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
 class TestAutomation(EditorTestSuite):
-    #global_extra_cmdline_args=["--regset=/Amazon/Preferences/EnablePrefabSystem=true"]
 
     class test_AxisAlignedBoxShape_ConfigurationWorks(EditorSingleTest):
         from .EditorScripts import TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges as test_module
+
+    class test_Terrain_SupportsPhysics(EditorSingleTest):
```

Comment:
Have you attempted to run these 2 tests together as EditorSharedTest so they're batched? I don't see any reason it wouldn't work, but maybe missed it.

Reply:
I'll test this and change if it works.

Reply:
Just tested and they work fine. I'll setup a quick PR in the morning to add these changes.

Reply:
Awesome!